tests/unit/TextUI/SourceMapperTest.php

```diff
@@ -24,55 +24,30 @@ final class SourceMapperTest extends TestCase
 {
     public static function provider(): Generator
     {
-        $fixtureDirectory = realpath(__DIR__ . '/../../_files/source-filter');
+        // temp remove
+        $fixtureDirectory = rtrim(self::fixturePath(''), '/');
 
         yield 'file included using file' => [
             [
-                $fixtureDirectory . DIRECTORY_SEPARATOR . 'a' . DIRECTORY_SEPARATOR . 'PrefixSuffix.php' => true,
+                self::fixturePath('a/PrefixSuffix.php') => true,
             ],
-            new Source(
-                null,
-                false,
-                FilterDirectoryCollection::fromArray([]),
-                FileCollection::fromArray(
-                    [
-                        new File($fixtureDirectory . '/a/PrefixSuffix.php'),
-                    ],
-                ),
-                FilterDirectoryCollection::fromArray([]),
-                FileCollection::fromArray([]),
-                false,
-                false,
-                false,
-                false,
-                false,
-                false,
-                false,
-                false,
-                false,
-                [
-                    'functions' => [],
-                    'methods'   => [],
-                ],
-                false,
-                false,
-                false,
+            self::createSource(
```

Comment:
I introduced a static factory with named parameters to make the test a bit easier to read / maintain